pkg/replay.go

```diff
@@ -55,9 +57,21 @@ func Replay(phasesStr string, silent, dryRun bool, timeout int) {
 
 	for scanner.Scan() {
 		req, err := unmarshalRequest(scanner.Bytes())
-
 		if err != nil {
-			panic(err)
+			if strictMode {
+				panic(err)
+			} else {
+				exitCode = 126
```

Comment:
@eugenepaniot no need for an `else` block here, we can save two lines of and a bit of indentation.

Reply:
done